src/layman/map/rest_workspace_test.py

```diff
@@ -302,12 +279,7 @@ def test_post_maps_complex():
 
     # assert another PATCH is not possible now
     with pytest.raises(LaymanError) as exc_info:
-        process_client.patch_workspace_map(workspace=workspace,
-                                           name=mapname,
-                                           title='abcd',
-                                           check_response_fn=empty_method_returns_true,
-                                           raise_if_not_complete=False,
-                                           )
+        process_client.patch_map_by_uuid(uuid=uuid_str)
```

Comment:
I think params `title`, `check_response_fn` and `raise_if_not_complete` should be preserved.